docs/How-to-test-the-application.md

```diff
@@ -0,0 +1,56 @@
+# How to test App:
+- Using Physical device and apk
+- Using Emulator and Apk
+- Directly Running in Android Studio ( Recommended )
+
+
+## For option one or two:
+
+Every time a new commit is merged into the develop branch an apk is released in the apk branch.To Navigate to apk Branch see here: 
+
+![Branch chnage](https://user-images.githubusercontent.com/16835617/92631953-8a52a900-f2ef-11ea-9e61-60692555283b.png)
+
+
+If you want to do manual testing on the application without setting up the development environment, you can download this app-debug.apk and install it on your Android device.
+
+![apk-branch](https://user-images.githubusercontent.com/16835617/92632334-a5251d80-f2ef-11ea-8d7d-007374cd882e.png)
+
+
+![apkInstall](https://user-images.githubusercontent.com/16835617/92632771-c4bc4600-f2ef-11ea-8d30-6618d990166d.png)
+
+
+For manually testing in physical device tick the option here :
+![Screenshot of Unknown install option ](https://static.apkpure.com/www/static/imgs/unknown_sources.jpg)
+
+Now click the apk to install !
+
+## For Testing on Emulator with APk :
+
+1. Now Open the project in Android Studio
+2. Wait untill all background process are finished 
+3. click the ADV button or click from already created devices in Android studio click 
+![advPress](https://user-images.githubusercontent.com/16835617/92633082-de5d8d80-f2ef-11ea-9593-ffb3646e8b55.png)
+4. For information on how to create an Emulator and Run it visit [Creating an ADV](https://developer.android.com/studio/run/managing-avds#createavd)
+5. Now Run the Emulator you have created 
+![RunADV](https://user-images.githubusercontent.com/16835617/92633141-f59c7b00-f2ef-11ea-9e3b-f686995fc57f.png)
+6. Navigate to the apk file downloaded 
+![apkAndEmulator](https://user-images.githubusercontent.com/16835617/92633190-0b11a500-f2f0-11ea-9c0b-2d4d7ee76162.png)
+7. Drag and Drop the Apk file into the Emulator.
+8. Now just look for the App in App tray and click to open.
+![gif](./images/gifDemo.gif)
```

Comment:
can you remove the gif from the commit and upload to github user content like the other images?

Reply:
No GitHub doesn't support gif that way

Reply:
@TreavVasu Can you trying to make this change now, as github has made some changes and can now even support videos?